services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,18 +43,39 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
-
```

Comment:
check if event is null

Reply:
In my new code, I checked if it the event points to a null pointer. I made the function return ERR_CODE_UNKNOWN if the event points to a null pointer or xQeueueSend() fails. Is this an okay approach?

Reply:
it's best to deal with each error case separately so that your error codes can be as specific as possible, so for example if event is NULL you would want to return `ERR_CODE_INVALID_ARG` or if the queue send fails you would want to return `ERR_CODE_QUEUE_FULL`